source/frontend/src/features/mapSideBar/acquisition/tabs/fileDetails/update/UpdateAcquisitionForm.tsx

```diff
@@ -266,6 +296,19 @@ const AcquisitionDetailSubForm: React.FC<{
             View={ContactInputView}
           ></ContactInputContainer>
         </SectionField>
+        {!isPersonSelected && (
+          <SectionField label="Primary contact" className="mt-4">
+            {primaryContacts.length > 0 ? (
```

Comment:
Looks like you are missing a requirement. If there is only one primary contact, it should be automatically selected. as per:

Screenshot 2: User cannot select different primary contact, if there is only ONE attached to linked organization